dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/inject/InjectInjectionObjectFactory.java

```diff
@@ -192,13 +196,19 @@ public Object run() {
         if (references.size() == 1) {
                 Object reference = references.get(0);
             if (TraceComponent.isAnyTracingEnabled() && tc.isEntryEnabled()) {
-                Tr.exit(tc, "getObjectInstance", reference);
+                Tr.exit(tc, "getObjectInstance", Util.identity(reference));
             }
             return reference;
         } else {
             Object[] referencesArray = references.toArray(new Object[references.size()]);
             if (TraceComponent.isAnyTracingEnabled() && tc.isEntryEnabled()) {
-                Tr.exit(tc, "getObjectInstance", referencesArray);
+                StringBuilder exitStringBuilder = new StringBuilder("[");
+                for (Object object : referencesArray){ 
+                    exitStringBuilder.append(Util.identity(object));
+                    exitStringBuilder.append(", ");
+                }
+                String exitString = exitStringBuilder.toString();
+                Tr.exit(tc, "getObjectInstance", exitString.substring(0,exitString.length()-2) + "]");
```

Comment:
better to move the string manipulation logic outside of the Tr.exit call, add an extra line above